src/main/java/edu/iis/powp/command/FactoryCircle.java

```diff
@@ -0,0 +1,33 @@
+package edu.iis.powp.command;
+
+public class FactoryCircle {
+
+    public ComplexCommand draw() {

```

Comment:
The name is confusing. It is generation rather than drawing (call to execute draws)